website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -1,500 +1,104 @@
 ---
-title: Golang Based Operator Quickstart
-linkTitle: Quickstart
-weight: 10
+title: Quickstart for Go-based Operators
+linkTitle: QuickStart
+weight: 20
+description: A simple set of instructions that demonstrates the basics of setting up and running a Go-based operator.
 ---
 
-**NOTE:** For the SDK versions prior to `v0.19.0` please consult the [legacy docs][legacy_quickstart_doc] for the [legacy CLI][legacy_CLI] and project.
-
-This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
+This guide walks through an example of building a simple memcached-operator powered by Go using tools and libraries provided by the Operator SDK
```

Comment:
```suggestion
This guide walks through the steps necessary to quickly build and run a simple Go-based memcached operator.
```

Reply:
But that's _really_ close to the description still. Yeah, I'd say just remove this line. It's pretty extraneous now that we have the description.